generators/app/templates/app/libraries/ModelController.ts

```diff
@@ -90,7 +90,6 @@ export function parseWhere(req: Request): any {
   }
 
   // Merge with req.session.where (Useful for enforcing policies)
-  if (req.session == null) req.session = {};
```

Comment:
The `ModelController` class extends `Controller`. In the parent constructor, the session object is initialized if it doesn't exist, so this check can be safely removed because the session object always exists.